sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/key/JwksKeySource.java

```diff
@@ -89,6 +93,10 @@ public String toString() {
     return "JwksKeySource{" + "jwksUri='" + jwksUri + '\'' + '}';
   }
 
+  public String getJwksUri() {
+    return jwksUri;
+  }
+
```

Comment:
Probably the same as in [OpenIdProviderDiscoveryKeySource](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1030/files#r667886860):

As said below, I think it is not necessary to validate domains of the issuer in the token compared to the issuer of the configuration because we compare the exact required issuer. So it is not needed to expose the value here.

Maybe logging a warning if requiredIssuer and issuerUrl do not match the same domain on startup could be an alternative.